Styled-components not being added as dependency

Beagle React version: 1.0.0

## Steps To Reproduce

1. Create a React app using create-react-app
2. Add beagle-react library
3. Add basic beagle config to render a simple component
4. Run project

Link to code example:

<!--
  Please provide a CodeSandbox (https://codesandbox.io/s/new), a link to a
  repository on GitHub, or provide a minimal code example that reproduces the
  problem. You may provide a screenshot of the application if you think it is
  relevant to your bug report. Here are some tips for providing a minimal
  example: https://stackoverflow.com/help/mcve.
-->

## The current behavior

The following Error is shown after running a basic React project from create-react-app using the beagle-react library

Module not found: Can't resolve 'styled-components' in 'C:\Users\zupper\Documents\Projects\Pocs\beagle-react-app\node_modules\@zup-it\beagle-react\components\BeagleButton'

## The expected behavior

If we  install styled-components everything works fine. But as it is a dependency we should not have to install it separately

--- a/package.json
+++ b/package.json
@@ -29,7 +29,8 @@
   "dependencies": {
     "@zup-it/beagle-web": "1.0.1",
     "lodash": "^4.17.15",
-    "showdown": "^1.9.1"
+    "showdown": "^1.9.1",
+    "styled-components": "^5.1.1"
   },
   "peerDependencies": {
     "react": ">=16.8.0",
@@ -55,7 +56,6 @@
     "nodemon": "^2.0.4",
     "react": "^16.13.0",
     "react-dom": "^16.13.0",
-    "styled-components": "^5.1.0",
     "ts-jest": "25.2.1",
     "tslib": "^1.11.1",
     "typescript": "^3.8.3"

